### PR TITLE
Feature - Application Access Revoked Notifications

### PR DESCRIPTION
**Pull request by ZilvinasKucinskas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -592,6 +592,21 @@ Example payload:
 }
 ```
 
+#### Application access revoked
+
+User can revoke an application access at any given time without prior notice trough Fractal ID dashboard. Upon getting `application_access_revoked` notification, partner can be notified that the user has decided not to share his information from now on. It means that all access grants and access tokens are being revoked and partner won't be able to query this user anymore.
```
Comment (juliosantos) [inline README.md]:
```suggestion
User can revoke an application access at any given time without prior notice trough Fractal ID dashboard. Upon getting `application_access_revoked` notification, partner can be notified that the user has decided not to share his information from now on. It means that all access grants and access tokens are being revoked and partner won't be able to query this user anymore.

Note that, due to AML regulations, Fractal still has to make this information available upon Client request.
```

**Comment by ZilvinasKucinskas** [comment]
@juliosantos Sure, let's wait. I have added a special tag to the branch as well.

File: README.md [hunk 2]
```diff
@@ -592,6 +592,23 @@ Example payload:
 }
 ```
 
+#### Authorization revoked
+
+User can revoke an application access at any given time without prior notice trough Fractal ID dashboard. Upon getting `authorization_revoked` notification, partner can be notified that the user has decided not to share his information from now on. It means that all access grants and access tokens are being revoked and partner won't be able to query this user anymore.
```
Comment (hugopeixoto) [inline README.md]: Few changes:
- `trough -> through`
- `share his` -> `share their`
- "Upon getting `authorization_revoked` notification, partner can be notified that the user has decided not to share their information from now on" -> "Fractal ID will emit a `authorization_revoked` notification when this happens."
- `are being` -> `were`? By the time they get the notification, this should all have happened, right?

Comment (ZilvinasKucinskas) [reply README.md]: Updated, thanks for the input 💯

**Pull request merged**